book/P3C14_forests.qmd

```diff
@@ -0,0 +1,341 @@
+Therefore, to ensure the bootstrapped prediction accurately represents the underlying mixed probability distribution, the bootstrapped cumulative hazard function should be computed as:
+
+$$
+\hat{H}_{Boot}(\tau|\xx^*) = - \log (\hat{S}_{Boot}(\tau|\xx^*))
+$$ {#eq-surv-nelson-boot}
+
+Another practical consideration to take into account is how to average the survival probabilities over the decision trees as each individual Kaplan-Meier estimate may have been trained on different time points.
+This is overcome by recognising that the Kaplan-Meier estimation results in a piece-wise function that can be linearly interpolated between training data.
+@fig-surv-ranfor-bootstrap demonstrates this process for three decision trees (panel a), where the survival probability is calculated at all possible time points (panels b-c), and the average is computed with linear interpolation added between time-points (panel d).
+
+![Bootstrapping Kaplan-Meier estimators across three decision trees (red, blue, green). Panel a) shows the individual estimates, b) shows the time points to aggregate the trees over, c) is the predicted survival probability from each tree at the desired time points, and d) is the average survival probabilities connected by a step function.](Figures/forests/bootstrap.png){#fig-surv-ranfor-bootstrap fig-alt="Four panels with 't' on x-axis and 'S(t)' on y-axis. Panel a) shows three survival functions as piece-wise linearly decreasing step functions. Panel (b) shows the same but with vertical dotted lines added at all time-points. Panel (c) shows dots at each of the intersections between the vertical lines and the plotted decision trees. Panel (d) shows a single line with dots which are the average of the points in panel (c)."}
+
+Extensions to competing risks follow naturally using bootstrapped cause-specific cumulative incidence functions.
+
+<!-- FIXME - ANDREAS TO ADD DETAILS ON COMPETING RISKS -->
+
+
+
+#### Deterministic Predictions {.unnumbered .unlisted}
+
+As discussed in @sec-eval-det, predicting and evaluating survival times is a complex and fairly under-researched area.
+For RSFs, there is an inclination to estimate survival times based on the mean or median survival times of observations in terminal nodes, however this would lead to biased estimations.
+Therefore, research has tended to focus on relative risk predictions.
+
+As discussed, relative risks are arbitrary values where only the resulting rank comparing observations matters.
+In RSFs, each terminal node should be as homogeneous as possible, hence within a terminal node, the risk between observation should be the same.
+Hence, one could theoretically assign a constant risk value to all terminal nodes, compare these using some concordant type metric, and shuffle repeatedly until maximum concordance is achieved.
```

Comment:
I think this could be savely deleted without loosing much and just state the result from Ishwaran